dags/openshift_nightlies/manifest.yaml

```diff
@@ -56,11 +64,20 @@ platforms:
       providers:
       - name: aws
         profiles:
-        - default
-        - ovn
+        - sdn-common-cp
```

Comment:
can we just remove `common` from all the profiles? When combined with the version/platform the dag IDs become long and annoying in visualization tools. I'm more in favor of using those characters for spelling out control/data plane 

Also i don't think it needs to be a part of this PR necessarily but with the profiles using hypens the dag IDs are gonna look something like `4.9_aws_sdn-common-dp` or `4.9_aws_sdn-data-plane` which bugs my ocd

Reply:
sure. thoughts? @jtaleric